.github/PULL_REQUEST_TEMPLATE/gitlab.md

```diff
@@ -0,0 +1,56 @@
+<!-- 
+This is the PR template for upgrading the GitLab instance.
+-->
+
+Connected issue: #4014
+
+
+## Checklist
+
+
+### Author
+
+- [ ] Target branch is `dev`
+- [ ] Source branch matches `gitlab/yyyy-mm-dd`
+- [ ] PR title is `Update GitLab to major.minor.patch`
+- [ ] PR is connected to issue #4014 via Zenhub
+
+
+### Primary reviewer (after approval)
+
+- [ ] Verified background migrations for `gitlab-dev` are complete
+- [ ] Verified background migrations for `gitlab-prod` are complete
+- [ ] Actually approved the PR
+- [ ] Labeled PR as `no sandbox`
+- [ ] Moved ticket to *Approved* column
+- [ ] Assigned PR to current operator
+
+
+### Operator (before pushing merge the commit)
+
+- [ ] Pushed PR branch to GitHub
+- [ ] Added PR reference to merge commit title
+- [ ] Collected commit title tags in merge commit title
+- [ ] Pushed merge commit to GitHub
+
+
+### Operator (after pushing the merge commit)
+
+- [ ] Made announcement requested by author                         <sub>or PR description does not contain an announcement</sub>
```

Comment:
I don't think this applies here. And I think you've based this on a outdated version of the default checklist. If that's the case, you should start from scratch.